pkg/util/ready/ready.go

```diff
@@ -24,6 +27,26 @@ import (
 	apiregistrationv1client "k8s.io/kube-aggregator/pkg/client/clientset_generated/clientset/typed/apiregistration/v1"
 )
 
+// IsMatchingSyscallError returns true when the error is one of the Errno's in match
+// it deals with the different ways of wrapping the syscalls.
+func IsMatchingSyscallError(err error, match []syscall.Errno) bool {
```

Comment:
don't think pkg/util/ready is the right place for this - if nothing else, a new pkg/util/errors?
use `...syscall.Errno` instead of `[]syscall.Errno` to neaten up the call sites

Reply:
ok